lib/recording/RecordingEntrySession.js

```diff
@@ -369,16 +426,51 @@ class RecordingEntrySession {
 
     }
 
+    verifyFlavorSetMatchPlaylistFlavors(newEntryObj) {
+        let playlistFilePath = path.join(this.recordingSessionPath, 'playlist.json');
+        return qio.read(playlistFilePath)
+            .then((data) => {
+                let playlistJSON = JSON.parse(data);
```

Comment:
can't we use the playListgenerator class to laod the json?
if we decide to change the format in the future, this code will break